src/domains/ecommerce-common/03_storage.tf

```
@@ -35,31 +35,31 @@ resource "azurerm_private_endpoint" "storage_private_endpoint" {
 
   private_service_connection {
     name                           = "${local.project}-storage-private-service-connection"
-    private_connection_resource_id = module.ecommerce_storage.id
+    private_connection_resource_id = module.ecommerce_storage_transient.id
     is_manual_connection           = false
     subresource_names              = ["queue"]
   }
 
   tags = var.tags
 }
 
-module "ecommerce_storage" {
+module "ecommerce_storage_transient" {
   source = "git::https://github.com/pagopa/terraform-azurerm-v3.git//storage_account?ref=v6.7.0"
 
 
   name                            = replace("${local.project}-sa", "-", "")
```

I'd include `transient` here
```suggestion
  name                            = replace("${local.project}-transient-sa", "-", "")
```